app/lib/logic/cms.rb

```diff
@@ -60,7 +60,7 @@ def cms_toolbar_intro_visible?
       end
 
       def create_cms_assets
-        self.builtin_sections.create!(title: 'Root', system_name: 'root', parent_id: nil)
```

Comment:
Where are protected attributes defined? Why was `parent_id` added in the first place if it was protected? 🤔 It's a bit strange that this attribute is not set afterwards..

Reply:
Protected attributes are defined no where but this module is used in a model and there also nothing describes, One strange thing is parent_id: nil is passed so what is the sense of passing any attribute nil so I removed it.

Reply:
@josemigallas can we merge this?

Reply:
not  tested resetting DB let me test this way